compiler/one-cmds/tests/one-quantize_009.qconf.json

```diff
@@ -0,0 +1,43 @@
+// Quantization configuration file
```

Comment:
(just comment; not sure adding // comment in json is standard. I think the json lib that we use may handle this. if we change the lib for some reason, this maybe a problem. you can see this is in read background)

Reply:
I've added comment for readability although it is not json standard. But in another thought, it would be better to follow the standard, so that we can parse this file in another platform (ex: Python).